pkg/controller/catalog/server/k8s_handler.go

```diff
@@ -109,14 +101,14 @@ func (s *k8sHandler) serviceBindingCallback(e k8swatch.Event) error {
 }
 
 func (s *k8sHandler) serviceBrokerCallback(e k8swatch.Event) error {
-	var sb scmodel.ServiceBroker
+	var sb servicecatalog.Broker
 	err := util.TPRObjectToSCObject(e.Object, &sb)
 	if err != nil {
 		glog.Errorf("Failed to decode the received object %#v", err)
 		return err
 	}
 
-	if e.Type == "ADDED" || e.Type == "MODIFIED" {
+	if e.Type == k8swatch.Added {
```

Comment:
One nit - we should probably add a line that says that Update of a broker isn't supported yet.